test/backends/subdomain_services.spec.ts

```diff
@@ -1,21 +1,24 @@
 import { expect } from "chai";
-import { ghostProBlog, glitch, netlify, heroku } from "../../src/backends"
+import { squarespace, ghostProBlog, glitch, netlify, heroku } from "../../src/backends"
 import * as errors from "../../src/errors";
 import { normalizeOptions } from "../../src/backends/subdomain_service";
 
-const defs = [
+const defs: any[] = [
   { backend: ghostProBlog, tests: [
     { subdomain: "fly-io", hostname: 'fly.io', directory: "/articles/" },
     { subdomain: "demo" }
   ]},
   { backend: glitch, options: ["subdomain"], tests: [
-    {appName: "fly-example" },
+    { appName: "fly-example" },
   ]},
   { backend: heroku, tests: [
-    { appName: "example"}
+    { appName: "example" }
   ]},
-  { backend: netlify, options: ["subdomain", "directory"], tests: [{
-    subdomain: "example"}
+  { backend: netlify, options: ["subdomain", "directory"], tests: [
+    { subdomain: "example" }
+  ]},
+  { backend: squarespace, tests: [
+    { subdomain: "example", status: 302 }
```

Comment:
We probably need a test squarespace site that doesn't return a 302. `example.squarespace.com` is the only one I can find that does a redirect.